build/Helix/runtests.cmd

```diff
@@ -1,6 +1,13 @@
 robocopy %HELIX_CORRELATION_PAYLOAD% . /s /NP
 
-te MUXControls.Test.dll MUXControlsTestApp.appx IXMPTestApp.appx /enablewttlogging /unicodeOutput:false /sessionTimeout:0:15 /testtimeout:0:10 /screenCaptureOnError %* 
+reg add HKLM\Software\Policies\Microsoft\Windows\Appx /v AllowAllTrustedApps /t REG_DWORD /d 1 /f
+
+cd scripts
+powershell -Command Add-AppxPackage ../Microsoft.VCLibs.%testbuildplatform%.14.00.appx
+powershell -Command Add-AppxPackage ../Microsoft.NET.CoreRuntime.1.1.appx
```

Comment:
Can this be driven off the dependencies.txt file or just running over the appx packages in the dependencies folder? The reason I ask is that these file names can change between release/debug and platform or the package name might be entirely different (like debug might use .NET Core but might use .NET Native).

Reply:
Good plan. Let me try that.